docker-compose.yml

```diff
@@ -11,7 +11,7 @@
 #   actively develop Octobox, you can override this file by adding the following to
 #   a docker-compose.override.yml:
 #
-#   version '3'
+#   version: '3'
```

Comment:
i guess this could be in a separate PR, but this tripped me up when setting up

Reply:
Bounced into a new pr: https://github.com/octobox/octobox/pull/1166